projects/plugins/paypal-payment-buttons/readme.txt

```diff
@@ -1,70 +1,90 @@
+2. Then paste the code for the `<head>` into the first text box in the PayPal Payment Buttons block. (We’ll take care of placing this code into your post or page `<head>` only once.)
+3. Then paste the `<body>` code into the second text box in the PayPal Payment Buttons block.
 
-1. Some feature
-1. Another feature
-1. Something else about the plugin
+#### If you are using the Single Button:
 
-Unordered list:
+1. Click the Single Button option in the PayPal Payment Buttons block.
+2. Copy the single button code from the PayPal site and paste it into the text area of the block.
 
-* something
-* something else
-* third thing
+By repeating the process above, you can add as many PayPal Payment Buttons blocks to your page as you like.
 
-Here's a link to [WordPress](https://wordpress.org/ "Your favorite software") and one to [Markdown's Syntax Documentation][markdown syntax].
-Titles are optional, naturally.
+### Taking Payments with PayPal
+
+1. When a visitor clicks a PayPal payment button, a new window will open to guide them through the PayPal checkout process.
+2. Visitors can use an existing PayPal account or a credit or debit card to complete their purchase.
+3. All payments are credited to the PayPal business account that was used to create the payment button code.
+
+== Installation ==
 
-[markdown syntax]: http://daringfireball.net/projects/markdown/syntax
-"Markdown is what the parser uses to process much of the readme file"
+1. Upload the PayPal Payment Buttons plugin to your WordPress site.
+2. Activate the plugin through the 'Plugins' screen in WordPress.
+3. Add the PayPal Payment Buttons block to any post or page using the block editor.
+4. Configure your PayPal account settings within the block to start accepting payments.
 
-Markdown uses email style notation for blockquotes and I've been told:
-> Asterisks for *emphasis*. Double it up  for **strong**.
+== Frequently Asked Questions ==
+
+= Can I test with a sandbox account? =
+
+It is possible to perform test payments with the PayPal Payment Buttons block. To get started you would need to create a <a href="https://developer.paypal.com/home/">PayPal Developer account</a>. Once you are logged into your PayPal developer account, you can access or create new sandbox accounts. You will need to make note of your sandbox business account and personal account email addresses and passwords. Once you have this information, you would login to the <a href="https://www.sandbox.paypal.com/">PayPal Sandbox site</a> with the sandbox business account. Create a <a href="https://www.sandbox.paypal.com/ncp/buttons/create?utm_source=wp&at_code=wp">payment button on the PayPal sandbox site</a>. Follow the instructions above to add the payment button code to your PayPal Payment Buttons block. Publish the post or page that contains the block. Then use the sandbox personal account to complete the purchase. All successful test payments will show up in the business sandbox account on the PayPal Sandbox.
```

Comment:
You  can use markdown for the links ; that may be easier to read in these reviews.